docs/source/deprecation_page.rst

```diff
@@ -0,0 +1,234 @@
+Deprecation for Kokkos-3.x
+==========================
+
+
+
+Type aliases deprecated in Kokkos-3.7
+-------------------------------------
+``ActiveExecutionMemorySpace``, ``ActiveExecutionMemorySpace``, ``is_array_layout``, ``is_execution_policy``, ``is_execution_space``, ``is_memory_space``, ``is_memory_traits``, ``host_memory_space``, ``host_execution_space``, ``host_mirror_space``, ``is_space``, ``SpaceAccessibility``, ``ActiveExecutionMemorySpace``, ``ActiveExecutionMemorySpace``, ``ActiveExecutionMemorySpace``, ``Iterate``, ``MDRangePolicy``, ``Rank``
+
+
+Macros deprecated in Kokkos-3.7
+-------------------------------
+
+``KOKKOS_RESTRICT_EXECUTION_TO_(DATA_SPACE)``, ``HIP_SAFE_CALL(call)``
+
+
+Free-functions deprecated in Kokkos-3.7
+---------------------------------------
+
+.. list-table::  
+   :widths: 30 70
+   :header-rows: 1
+
+   * - Name 
+     - Where
+
+   * - .. code-block:: cpp 
+
+          std::vector<OpenMP> OpenMP::partition(...) {
+            return std::vector<OpenMP>(1); 
+          }
+
+     - OpenMP
+
+   * - .. code-block:: cpp
+
+          OpenMP OpenMP::create_instance(...) {
+            return OpenMP(); 
+          }
+
+     - OpenMP
+
+   * - .. code-block:: cpp
+
+          void OpenMP::partition_master(F const& f,
+                                        int num_partitions,
+                                        int partition_size)
+
+     - OpenMP (Kokkos_OpenMP_Instance.hpp)
+
+   * - .. code-block:: cpp
+
+          void Experimental::HIPSpace::access_error()
+
+     - ``namespace Kokkos`` (Kokkos_HIP_Space.cpp)
+
+   * - .. code-block:: cpp
+
+          void Experimental::HIPSpace::access_error(const void* const)
+
+     - ``namespace Kokkos`` (Kokkos_HIP_Space.cpp)
+
+   * - ..  code-block:: cpp
+
+           inline void hip_internal_safe_call_deprecated
+
+     - ``namespace Kokkos::Impl`` (Kokkos_HIP_Error.hpp)
+
+
+Member functions deprecated in Kokkos-3.7
+------------------------------------------
+
+.. list-table::  
+   :widths: 70 30
+   :header-rows: 1
+
+   * - Method name
+     - Class
+
+   * - ``static void partition_master(F const& f,
+                                      int requested_num_partitions = 0,
+                                      int requested_partition_size = 0)``
+     - ``class OpenMP`` (Kokkos_OpenMP.hpp)
+
+   * - ``static void validate_partition(const int nthreads,
+                                         int& num_partitions,
+                                         int& partition_size)``
```

Comment:
The rendering is weird. I would just drop the arguments.
```suggestion
   * - ``static void partition_master()``
     - ``class OpenMP`` (Kokkos_OpenMP.hpp)

   * - ``static void validate_partition()``
```

Reply:
Done